Sources/LLBBuildSystem/Generated/BuildSystem/Evaluation/artifact.pb.swift-e

```diff
@@ -0,0 +1,192 @@
+// DO NOT EDIT.
+// swift-format-ignore-file
```

Comment:
this seems like an intermediate artifact, should be removed?

Reply:
🤷‍♂️ don't know where that comes from, but it keeps coming back when running generate

Reply:
ah, artifact from `sed`.   Fixed it up.